script/deploy/deploySingleContract.sh

```
@@ -116,7 +116,12 @@ deploySingleContract() {
   fi
 
   # check if deploy script exists
-  local FULL_SCRIPT_PATH=""$DEPLOY_SCRIPT_DIRECTORY""$SCRIPT"".s.sol""
+  if [[ $NETWORK == "zksync" ]]; then
+    FULL_SCRIPT_PATH=""$DEPLOY_SCRIPT_DIRECTORY""$SCRIPT"".zkysync.s.sol""
+  elif
```

Shellcheck (0.10.0)

[error] Did you forget the 'then' for this 'elif'?

(SC1049)